Dockerfile-alpine.template

```diff
@@ -43,4 +43,20 @@ RUN adduser -D -u 1000 node \
     && rm -Rf "node-v$NODE_VERSION" \
     && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
 
+ENV YARN_VERSION 0.0.0
```

Comment:
@Starefossen take a look at the `NODE_VERSION` variable above - it's also set to `0.0.0` in the templates.